src/types.ts

```diff
@@ -152,7 +152,7 @@ export interface EtherTokenContract extends ContractInstance {
     withdraw: (amount: BigNumber.BigNumber, txOpts: TxOpts) => Promise<void>;
 }
 
-export interface ProxyContract extends ContractInstance {
+export interface TokenTransferProxyContract extends ContractInstance {
```

Comment:
Don't forget to update the `typeToSection` mapping in website types to `tokenTransferProxyWrapper`.